### PR TITLE
adjust ncurses

### PR DESCRIPTION
**Pull request by lovedheart**
Compile libncurses and libncursesw as a compact package and adjust dependencies on some packages

### CODE REVIEW

File: packages/aspell.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Aspell < Package
   })
 
   depends_on 'ruby' unless File.exists? "#{CREW_PREFIX}/bin/ruby"
-  depends_on 'ncursesw'
+  depends_on 'ncurses'
```
Comment (uberhacker) [inline packages/aspell.rb]: If you leave the `ncursesw` dependency, you get both `ncurses` and `ncursesw`.  Any reason not to have `ncursesw`?

Comment (lovedheart) [reply packages/aspell.rb]: the modified ncurses have both ncurses and ncursesw

File: packages/ncurses.rb [hunk 2]
```diff
@@ -3,38 +3,68 @@
...
+             "--with-cxx-shared",
+             "--without-debug",
+             "--enable-pc-files",
+             "--with-pkg-config-libdir=#{CREW_LIB_PREFIX}/pkgconfig",
+             "--disable-db-install",
+             "--without-manpages",
+             "--without-progs",
+             "--without-tack"
+      system "make"
+    end
```
Comment (uberhacker) [inline packages/ncurses.rb]: Oh, I see what you are doing now.  You want to combine ncursesw in with the ncurses package?

Comment (lovedheart) [reply packages/ncurses.rb]: Yes.

Comment (lovedheart) [reply packages/ncurses.rb]: Since they come from the same source code

Comment (uberhacker) [reply packages/ncurses.rb]: Yes, that makes a lot of sense.  I like the way you think. :)

File: packages/ncurses.rb [hunk 3]
```diff
@@ -3,38 +3,68 @@
...
-     armv7l: '4e33d4fc68ea5b47bae07aa710b1f801fd80ba85a02c0060c6f157100acadaf4',
-       i686: '16a5f5c03a4e9591146dc56a1ed4b017c09ad9d72da38d104ce282a3d25ef9fd',
-     x86_64: '1eab39eee49e16799ad61def29e69c50a7ea39b04a8da2ff4f4ada033e81273c',
+    aarch64: 'a12e70f779b2375912a5373b5f320c7b4bef8ddcba5144709e2aed67f8011afe',
+     armv7l: 'a12e70f779b2375912a5373b5f320c7b4bef8ddcba5144709e2aed67f8011afe',
+       i686: 'a89d5d79d91c8717cfecf7ebbebae6479f8cded2884798643d061605e449c1df',
+     x86_64: 'fcaf728604029920a14f27f64f498dfc8a3586bc0426ca6a890adf925e39e204',
   })
 
   depends_on 'diffutils' => :build
```
Comment (uberhacker) [inline packages/ncurses.rb]: Remove since in core?

**Pull request merged**